create_data_image.sh

```diff
@@ -1,25 +1,14 @@
 #!/bin/bash
 
-# Currently everything runs as root, but that is going to change.
-USER_UID=0
-USER_GID=0
-
-source ./su_command.sh
-
 TOTAL_SIZE=0
 
 echo "Gathering applications..."
-APPS=
-for app in apps/*
-do
-	if test -f $app
-	then
-		APPS="${APPS} $app"
-		SIZE=$(stat -Lc %s ${app})
+for app in apps/*; do
+	if test -f "$app";	then
+		SIZE="$(stat -Lc %s "$app")"
 		TOTAL_SIZE=$((${TOTAL_SIZE} + ${SIZE}))
 		echo "app: ${app} - $((${SIZE} / 1024)) kB"
-	elif [ $app != "apps/*" ]
-	then
+	else
```

Comment:
What is the motivation for this change? Won't it break when the `apps` dir is empty or non-existent?

Reply:
Resolving obsolete comment